ensime-model.el

```diff
@@ -30,6 +30,12 @@
 (defun ensime-symbol-decl-pos (sym)
   (plist-get sym :decl-pos))
 
+(defun ensime-symbol-name (sym)
+  (plist-get sym :name))
+
+(defun ensime-symbol-local-name (sym)
```

Comment:
I'm leaning against these little helpers, and instead preferring common-lisp style destructuring of the contents.